django_ipam/tests/base/test_api.py

```diff
@@ -1,28 +1,71 @@
+import json
+
+from django.contrib.auth.models import User
 from django.urls import reverse
 
 from .base import CreateModelsMixin
 
 
 class BaseTestApi(CreateModelsMixin):
-    def test_ipv4_iprequest_api(self):
+    def test_ipv4_get_avaialble_api(self):
         subnet = self._create_subnet(dict(subnet="10.0.0.0/24"))
         self._create_ipaddress(dict(ip_address="10.0.0.1", subnet=subnet))
 
-        response = self.client.post(reverse('ipam:request_ip', args=(subnet.id,)))
+        response = self.client.get(reverse('ipam:get_first_available_ip', args=(subnet.id,)))
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, '10.0.0.2')
 
-    def test_ipv6_iprequest_api(self):
+    def test_ipv6_get_avaialble_api(self):
         subnet = self._create_subnet(dict(subnet="fdb6:21b:a477::9f7/64"))
         self._create_ipaddress(dict(ip_address="fdb6:21b:a477::1", subnet=subnet))
 
-        response = self.client.post(reverse('ipam:request_ip', args=(subnet.id,)))
+        response = self.client.get(reverse('ipam:get_first_available_ip', args=(subnet.id,)))
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, 'fdb6:21b:a477::2')
 
-    def test_unavailable_iprequest(self):
+    def test_unavailable_ip(self):
         subnet = self._create_subnet(dict(subnet="10.0.0.0/32", description="Sample Subnet"))
 
-        response = self.client.post(reverse('ipam:request_ip', args=(subnet.id,)))
+        response = self.client.get(reverse('ipam:get_first_available_ip', args=(subnet.id,)))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, '')
```

Comment:
hey wait a moment, I think this should be:

```python
self.assertEqual(response.data, None)
```
I mean aren't we responding with `None` why should that be serialized to an empty string?
And why would you be using `assertContains`, which checks `response.content` which is not appropiate for a JSON response?